### DISCUSSION TITLE
Add header control to RewritePlugin

### DISCUSSION THREAD

**Discussion by 30c4** [dotnet/dev-proxy#1217] [Q&A]
I am currently in a situation where requests are gated by headers. 
Right now I am using a browser extension that modifies the headers but I was wondering if this is also possible within dev-proxy. 

Is this something that would fit in the RewritePlugin or maybe a completely new plugin?

**Comment by waldekmastykarz** [comment]
Can you please explain a bit more what you'd need? 🙂

**Comment by waldekmastykarz** [comment]
Closing due to lack of further comments

**Comment by canicad** [comment]
Similar to the OP, I am also looking to add a header to http requests intercepted by dev proxy. 
I know this can be done by writing a custom plugin, but the question is if its possible using an existing plugin (e.g. RewritePlugin) or other OOB functionality?
